Userland/Libraries/LibJS/Bytecode/ASTCodegen.cpp

```diff
@@ -129,7 +129,7 @@ Bytecode::CodeGenerationErrorOr<void> ScopeNode::generate_bytecode(Bytecode::Gen
             // Note: Already done in step iv. above.
 
             // 4. Insert d as the first element of functionsToInitialize.
-            functions_to_initialize.prepend(function);
+            functions_to_initialize.append(function);
```

Comment:
Why do we use append now?

Reply:
Should have seen that coming...
To quote the PR description:
> Also changes bytecode to be consistent with AST and the eval AO, while saving a good amount of memmoves through avoiding preprend

Shall I add another comment, or mention this in the commit description?

Reply:
I stopped reading the footnote after "to be consistent", please don't put critical information only into a PR description. Assume that someone will look at the code without ever having seen the PR description.

Reply:
Well it also has
```cpp
// 16. For each Parse Node f of functionsToInitialize, do
// Note: We used append in place of prepend, so we have to iterate in reverse
```
90 lines down like the others